packages/network/test/index.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { beforeAll, afterAll, afterEach, expect, test, assert } from 'vitest'
+
+import { setupServer } from 'msw/node'
+import { rest } from 'msw'
+
+import { Fetch, buildFetch } from '../src';
+
+export const restHandlers = [
+    rest.get('http://www.foo.com/resource', (req, res, ctx) => {
```

Comment:
minor nit but use `example.com`

The very point of `example.com` is for this kind of thing.

Reply:
example.com is a real website, which can be providing false positives. I will change this to utilize example.com.